app/server.go

```diff
@@ -117,7 +117,8 @@ type Server struct {
 
 	phase2PermissionsMigrationComplete bool
 
-	HTTPService httpservice.HTTPService
+	HTTPService            httpservice.HTTPService
+	pushNotificationClient *http.Client // TODO: move this to it's own package
```

Comment:
why the todo? what's blocking from moving it now?

Reply:
It's just a considerable effort that needs a lot of mana. It's much later in my plan of things to do.